plugins/woocommerce/src/Admin/Features/Navigation/Init.php

```diff
@@ -36,7 +36,9 @@ class Init {
 	public function __construct() {
 		add_action( 'update_option_' . self::TOGGLE_OPTION_NAME, array( $this, 'reload_page_on_toggle' ), 10, 2 );
 		add_action( 'woocommerce_settings_saved', array( $this, 'maybe_reload_page' ) );
-		add_action( 'admin_enqueue_scripts', array( $this, 'maybe_enqueue_opt_out_scripts' ) );
+
+		// We are disabling the opt-out modal as the feature development is on hold.
```

Comment:
<img width="551" alt="Screenshot 2023-01-23 at 11 47 18" src="https://user-images.githubusercontent.com/13561163/214030424-c652a94f-fe81-4389-825f-66fef630b587.png">

Changing the option value triggers the modal for the users with the navigation enabled.